pages/collections/_type/_.vue

```diff
@@ -261,6 +277,24 @@
           values: [title],
           code: null
         };
+      },
+      relatedLinkGen(item) {
+        let id = '',
+            name = '';
+        if (typeof item.id === 'undefined') {
+          id = item.identifier;
+          name = item.name;
+        } else {
+          id = item.id;
+          name = item.prefLabel[this.$i18n.locale];
```

Comment:
For the URL slugs, we always use English, so change this to `item.prefLabel.en`.

Otherwise, when on http://localhost:3000/de/collections/topic/221-watercolor-painting, following the link to "Zeichnung (Kunst)" takes you first to http://localhost:3000/de/collections/topic/35-zeichnung-(kunst), which then redirects to http://localhost:3000/de/collections/topic/35-drawing.

Use the English slug here and that redirect is avoided.

Reply:
sorry my mistake